docs/nodes/bridge-node.mdx

```diff
@@ -100,6 +139,37 @@ celestia bridge start --core.ip <ip-address> --core.grpc.port <port>
 
 If you need a list of RPC endpoints to connect to, you can check from the list [here](./mamaki-testnet.md#rpc-endpoints)
 
+Here is an example of starting the bridge node:
+
+```sh
+celestia bridge start --core.ip https://limani.celestia-devops.dev/ --core.grpc.port 9090
+```
+
+</TabItem>
+<TabItem value="mamaki" label="Mamaki">
+
+Start the Bridge Node with a connection to a validator node's gRPC endpoint
+(which is usually exposed on port 9090):
+
+```sh
+celestia bridge start --core.remote <ip-address>:<port>
+```
+
+> NOTE: The `--core.remote` port defaults to 9090, so if you do not specify it in the command line, it will default to that port. You can use the flag to specify another port if you prefer.
+
+If you need a list of RPC endpoints to connect to, you can check from the list [here](./mamaki-testnet.md#rpc-endpoints)
+
+Here is an example of starting the bridge node:
+
+```sh
+celestia bridge start --core.remote https://rpc-mamaki.pops.one:26657
+```
+
+</TabItem>
+</Tabs>
+
+---
```

Comment:
What's the reasoning behind having these three dashes here

Reply:
<img width="768" alt="Screenshot 2022-11-22 at 8 19 48 PM" src="https://user-images.githubusercontent.com/46639943/203452278-a6a55ad0-db2b-4c26-8d85-31ec1775162e.png">

it adds a divider below the section that is toggled with the tabs, but can definitely be removed